src/vxworks/mod.rs

```diff
@@ -1340,13 +1310,25 @@ extern "C" {
         filename: *const ::c_char,
         times: *const ::timeval,
     ) -> ::c_int;
+
+    #[link_name = "_rtld_dlopen"]
     pub fn dlopen(filename: *const ::c_char, flag: ::c_int) -> *mut ::c_void;
+
+    #[link_name = "_rtld_dlerror"]
     pub fn dlerror() -> *mut ::c_char;
+
+    #[link_name = "_rtld_dlsym"]
     pub fn dlsym(
         handle: *mut ::c_void,
         symbol: *const ::c_char,
     ) -> *mut ::c_void;
+
+    #[link_name = "_rtld_dlclose"]
     pub fn dlclose(handle: *mut ::c_void) -> ::c_int;
+
+    #[link_name = "_rtld_dladdr"]
+    pub fn dladdr(addr: *mut ::c_void, info: *mut Dl_info) -> ::c_int;
+    
```

Comment:
src/vxworks/mod.rs:1331 - trailing whitespace

I guess you didn't run "sh ci/style.sh".

Reply:
Sorry I'm setting a very bad example with this untested change :( Thanks for your diligence.

Ran ci/style.sh and ci/build.sh

Reply:
ci/build.sh is still running so please wait until I report back ....

Reply:
OK it finished and passed. Pity we can't just run it for the VxWorks targets that come right at the end of the list, the rest are a waste of time unless we change some of the common code!

Reply:
I have simplified version that only run vxworks targets:
/folk/prj-rust-dev/bpang/wr-rust/scripts/test/vxbuild.sh

And I am thinking about to add a "FILTER" to script build.sh so we can choose which target(s) to run.